document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -79,7 +83,15 @@ public DocumentInfoResponse getDocumentInfo(DocumentInfoRequest documentInfoRequ
             }
         }
 
-        return null;
+        try {
+            return smallFullAccountsDataHandler.getSmallFullAccountsData(transaction, resourceLink, requestId);
```

Comment:
This logic could be refactored to be more readable. Psuedo code:
```
try {
    if (isAccounts(resourceLink)) {
        return abridgedHandler.getData()
    } else {
        return smallFullAccountsHandler.getData()
    }
} catch (HandlerException e) {
    // Logging
    throw new DocumentInfoException
}
```
It's a little more concise to have a single catch point. 

I'd also recommend renaming the `isAccounts` method to something like `isAbridgedAccounts`. After all, a set of small full accounts _are_ accounts! (I know that was done in a much earlier PR but could still use a refactor).

Reply:
done